controllers/imagerepository_controller.go

```diff
@@ -319,6 +364,11 @@ func (r *ImageRepositoryReconciler) ProvisionImageRepository(ctx context.Context
 		}
 	}
 
+	var notificationStatus []imagerepositoryv1alpha1.NotificationStatus
+	if notificationStatus, err = r.AddNotifications(ctx, imageRepository); err != nil {
```

Comment:
Do we want to add the notifications on image repository creation only or at any point?

Reply:
For the current usecase that we have it is fine to create a notification on CR creation.

Reply:
It has to be implemented in follow up PR.